console/src/components/CompactAppList/CompactAppList.js

```diff
@@ -0,0 +1,35 @@
+import kind from '@enact/core/kind';
+import {Row} from '@enact/ui/Layout';
+import React from 'react';
+import AppIconCell from '../AppIconCell';
+
+const CompactAppList = kind({
+	name: 'CompactAppList',
+
+	styles: {
+		className: 'compact'
```

Comment:
Set the `className` to `compactAppList`, and include the `css` value above `className`.

Reply:
i don't have css for this now.

Reply:
Right. Good point about the `css`. If the `'compact'` name isn't going to be changed, please remove the styles section.